src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json

```diff
@@ -1,590 +1,846 @@
+                                "Value": "Full apprenticeship frameworks",
+                                "HintText": ""
+                            },
+                            {
+                                "Label": "Part apprenticeship standards",
+                                "Value": "Part apprenticeship standards",
+                                "HintText": ""
+                            },
+                            {
+                                "Label": "Part apprenticeship frameworks",
+                                "Value": "Part apprenticeship frameworks",
+                                "HintText": ""
+                            },
+                            {
+                                "Label": "Functional skills",
+                                "Value": "Functional skills",
+                                "HintText": ""
+                            }
+                        ],
+                        "Validations": [
+                            {
+                                "Name": "Required",
+                                "Value": "",
+                                "ErrorMessage": "Tell us what type of apprenticeship training your organisation will offer"
+                            }
+                        ]
+                    },
+                    "Order": null
+                }
+            ],
+            "Complete": false,
+            "AllowMultipleAnswers": false,
+            "Order": null,
+            "Active": true,
+            "Visible": true,
+            "Feedback": null,
+            "HasFeedback": false,
+            "NotRequiredConditions": [
+                {
+                    "Field": "Apply-ProviderRoute",
+                    "IsOneOf": [
+                        "1",
+                        "2"
+                    ]
+                }
+            ],
+            "Next": [
+                {
+                    "Action": "NextPage",
+                    "ReturnId": "6200",
+                    "Conditions": [
+                        {
+                            "QuestionId": "PAT-23",
+                            "QuestionTag": null,
+                            "Contains": "Full apprenticeship standards"
+                        }
+                    ],
+                    "ConditionMet": false
+                },
+                {
+                    "Action": "NextPage",
+                    "ReturnId": "6200",
+                    "Conditions": [
+                        {
+                            "QuestionId": "PAT-23",
+                            "QuestionTag": null,
+                            "Contains": "Part apprenticeship standards"
+                        }
+                    ],
+                    "ConditionMet": false
+                },
+                {
+                    "Action": "NextPage",
+                    "ReturnId": "6060",
+                    "Conditions": [
```

Comment:
As above - how will this condition be triggered if there's one above that goes to page 6200?

Reply:
the one above has different words in the contains